### DISCUSSION TITLE
What do I do about all this bloat in next 12?

### DISCUSSION THREAD

**Discussion by rcabre95** [vercel/next.js#36788] [Help]
I have been using next 11 for about 2 years now, and I just started using Next 12 a few days ago. The main thing I noticed is that all these new files keep updating themselves and new ones are created within the .next/ directory. Mostly they're .webpack.hot-update.json files. What are these for? Can i delete them? How do I stop them from being created?
Same questions for the "Trace" file. 

Thanks in advance
![Screen Shot 2022-05-09 at 3 34 35 PM](https://user-images.githubusercontent.com/48636832/167493121-01c5e056-9a23-4d63-9504-8133301bccb7.png)

**Comment by ijjk** [comment]
Hi, these files are used for HMR and are only needed in development mode, when a production build is created with `next build` they will be cleared.
